cdc/owner.go

```diff
@@ -1089,6 +1139,11 @@ func (o *Owner) handleAdminJob(ctx context.Context) error {
 			if err != nil {
 				return errors.Trace(err)
 			}
+			// refuse to resume adminJob that checkPointTs lag behind gcSafePoint
+			if o.minGCSafePointCache.ts != 0 && cfInfo.StartTs < o.minGCSafePointCache.ts {
+				log.Warn("invalid admin job, changefeed checkpoint lag behind gcSafePoint", zap.String("changefeed", job.CfID))
```

Comment:
It would be more user-friendly if the error message in changefeed status were updated. Otherwise, the user might not understand what is wrong.